doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -262,15 +262,16 @@ Clone the Helm installer directory using the following command:
 
 Navigate to the
 ``amazon-sagemaker-operator-for-k8s/hack/charts/installer`` folder. Edit
-the \ ``values.yaml`` file, which includes high-level parameters for the
-Chart. Replace the ARN here with the ARN for the OIDC-based role you’ve
+the \ ``rolebased/values.yaml`` file, which includes high-level parameters for the
+Chart. Replace the role Arn here with the Arn for the OIDC-based role you’ve
```

Comment:
We use ARN when referring to the Amazon Resource Name. For reference: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html#Conditions_ARN

Reply:
Even within that document, the variable names refer to it as "Arn". I get confused as to when to use which capitalisation. I suppose it's "ARN" in documentation and "Arn" in code?

Reply:
In documentation, we want to use Amazon Resource Name for the first usage, and ARN for subsequent usage. For code, the best practice is to conform to the standards being used.